srcpkgs/bzip2/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'bzip2'
 pkgname=bzip2
-version=1.0.7
+version=1.0.8
 revision=1
 bootstrap=yes
 short_desc="Freely available, patent free, high-quality data compressor"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
-license="bzip2-1.0.6"
+license="bzip2-1.0.8"
```

Comment:
1.0.7 had same license as 1.0.6. Did 1.0.8 changed that? If not, leave as-is, because bzip2-1.0.6 is included in SPDX and bzip2-1.0.8 will never be.

Reply:
```
temp: diff bzip2-1.0.6/LICENSE bzip2-1.0.7/LICENSE
39,40c39,40
< Julian Seward, jseward@bzip.org
< bzip2/libbzip2 version 1.0.6 of 6 September 2010
---
> Julian Seward, jseward@acm.org
> bzip2/libbzip2 version 1.0.7 of 27 June 2019


temp: diff bzip2-1.0.7/LICENSE bzip2-1.0.8/LICENSE
5c5
< documentation, are copyright (C) 1996-2010 Julian R Seward.  All
---
> documentation, are copyright (C) 1996-2019 Julian R Seward.  All
40c40
< bzip2/libbzip2 version 1.0.7 of 27 June 2019
---
> bzip2/libbzip2 version 1.0.8 of 13 July 2019
```

Reply:
I would still say it's same, we do not mark MIT license by different authors different.
SPDX bzip2-1.0.5 and bzip2-1.0.6 differ in patents note.

Reply:
So I will change it back to 1.0.6 tomorrow. Thanks. :)